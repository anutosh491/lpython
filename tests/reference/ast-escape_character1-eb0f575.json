{
    "basename": "ast-escape_character1-eb0f575",
    "cmd": "lpython --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/escape_character1.py",
    "infile_hash": "1eb54c099258db254defa83efbf30345041207d7a90b9a56a679d20c",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-escape_character1-eb0f575.stdout",
    "stdout_hash": "f250e7babfee60465b981012169ea3c497849635ad9757df9ffcab77",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}