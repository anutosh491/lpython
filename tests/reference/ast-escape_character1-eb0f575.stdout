(Module [(FunctionDef testEscapeCharacters ([] [] [] [] [] [] []) [(AnnAssign (Name s1 Store) (Name str Load) (ConstantStr "Hello\tWorld" ()) 1) (AnnAssign (Name s2 Store) (Name str Load) (ConstantStr "Hello    \bWorld" ()) 1) (AnnAssign (Name s3 Store) (Name str Load) (ConstantStr "Hello\vWorld" ()) 1) (Expr (Call (Name print Load) [(Name s1 Load)] [])) (Expr (Call (Name print Load) [(Name s2 Load)] [])) (Expr (Call (Name print Load) [(Name s3 Load)] []))] [] () ()) (FunctionDef main0 ([] [] [] [] [] [] []) [(Expr (Call (Name testEscapeCharacters Load) [] []))] [] () ()) (Expr (Call (Name main0 Load) [] []))] [])
